sbioapputils/app_runner/dev_utils.py

```diff
@@ -365,9 +368,83 @@ def payload_from_folder(folder_loc):
 
     if len(images) > 0:
         results_for_payload['images'] = _generate_file_dict(images)
+    else:
+        results_for_payload['images'] = {}
```

Comment:
we can just return empty dict from _generate_file_dict function if there are images. cause the function says in the name that it's gonna return a dict

Reply:
same for table and figures